src/dcap_provider.cpp

```diff
@@ -714,3 +721,112 @@ extern "C" sgx_plat_error_t sgx_ql_set_logging_function(
     logger_callback = logger;
     return SGX_PLAT_ERROR_OK;
 }
+
+extern "C" sgx_plat_error_t sgx_get_qe_identity_info(
+    sgx_qe_identity_info_t** pp_qe_identity_info)
+{
+    std::string intel_cert_base_url = INTEL_QE_IDENTITY_URL;
+    sgx_qe_identity_info_t *p_qe_identity_info = NULL;
+    char* buffer = nullptr;
+
+    if (!pp_qe_identity_info)
+    {
+        log(SGX_QL_LOG_ERROR, "Invalid parameter pp_qe_identity_info"); 
+        return SGX_PLAT_ERROR_INVALID_PARAMETER;
+    }
+
+    try 
+    {
+        std::vector<uint8_t> identity_info;
+        std::string issuer_chain;
+        size_t total_buffer_size = 0;
+
+        const auto curl = curl_easy::create(intel_cert_base_url, CYBERTRUST_ROOT_CERT);
+        log(SGX_QL_LOG_INFO, "Fetching QE Identity from remote server: '%s'.", intel_cert_base_url.c_str());
+        curl->perform();
+
+        // content type

```

Comment:
Why bother with getting content type? It's not used for anything.

Reply:
It was there for debugging purpose. I will remove it.